### DISCUSSION TITLE
SMB plugin for Pods running on Tanzu Kubernetes

### DISCUSSION THREAD

**Discussion by djciaro** [kubernetes-csi/csi-driver-smb#602] [General] [answered]
Hello,

To setup an access from a Container pod (Alpine image) to a windows share SMB someone suggested the CSI driver. However our Pod runs on a Tanzu Kubernetes cluster. vSphere already has CSI ( Container Storage Interface (CSI) for vSphere) which is used by the Supervisor cluster.
Is there a way to use the smb plugin leveraging the CSI provided by vSphere?
for example only the smbplugin itself? 
Many thanks

Ciarán

**Comment by andyzhangx** [comment, marked as answer]
that's vSphere CSI driver, you need to install this SMB CSI driver to access smb share. CSI driver is a k8s storage driver which implements Container Storage Interface, while different CSI driver implements different interfaces, they are different driver.
